### PR TITLE
Add script to post-process marine eva files

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
Currently this takes a directory of yaml files from gen_eva_obs_yaml.py and the template marine_gdas_plots.yaml and the diag file directory, corrects the yamls with the proper diag file paths, and then outputs to another directory. It is anticipated that this will perform other, similar tasks.

File added:
ush/eva/marine_eva_post.py

### CODE REVIEW

File: ush/eva/marine_eva_post.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import logging
+import os
+import socket
+import yaml
+import glob
+
+
+def marine_eva_post(inputdir, outputdir, diagdir):
+    logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+    inputyamls = glob.glob(os.path.join(inputdir, '*.yaml'))
```
Comment (guillaumevernieres) [inline ush/eva/marine_eva_post.py]: Let's make that function operate on 1 yaml only, while diagdir is going to be common to all, that isn't true for the bounds for example. Simpler to call this function with specific argument for a specific obs type.

**Review by guillaumevernieres — COMMENTED** [review]
Getting there @AndrewEichmann-NOAA . Just one minor comment.

File: ush/eva/marine_eva_post.py (hunk 1 above)
Comment (AndrewEichmann-NOAA) [reply ush/eva/marine_eva_post.py]: Ok, sounds safer

**Review by guillaumevernieres — COMMENTED** [review]
looks good, but I'm tempted to ask for the addition of a bit of testing ...

**Comment by AndrewEichmann-NOAA** [comment]
That might be a story for different PR

**Review by guillaumevernieres — APPROVED** [review]
:tada:

**Pull request merged**